### PR TITLE
Fix Javadocs to be consistent with UG/DG

### LINKED ISSUE #201: Ensure consistency between UG/DG and JavaDocs
The following commands should be checked for consistency between the UG and JavaDocs:

- [ ] `AddCommand`
- [ ] `ClearCommand`
- [ ] `DeleteCommand`
- [ ] `EditCommand`
- [ ] `ExitCommand`
- [ ] `FCallCommand`
- [ ] `FindCommand`
- [ ] `HelpCommand`
- [ ] `ListCommand`
- [ ] `SCallCommand`
- [ ] `ScheduleCommand`
- [ ] `SessionCommand`
- [ ] `SortCommand`
- [ ] `TShiftCommand`

### PR DESCRIPTION
**Pull request by luffingluffy**
Closes #201.

### CODE REVIEW

File: src/main/java/seedu/track2gather/logic/commands/SCallCommand.java [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
+ * Updates a person as successfully called in the current SHN enforcement session.
  */
 public class SCallCommand extends Command {
 
     public static final String COMMAND_WORD = "scall";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Indicates that the person has been successfully called\n"
+            + ": Updates a person as successfully called in the current SHN enforcement session.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
```
Comment (jennibearduit) [inline src/main/java/seedu/track2gather/logic/commands/SCallCommand.java:22]: Same as `FCall`

File: src/main/java/seedu/track2gather/logic/commands/FindCommand.java [hunk 2]
```diff
@@ -14,19 +14,17 @@
...
+ * Finds person(s) based on the field specified by the user.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons who fulfills the "
-            + "criteria specified by the user.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds person(s) based on the field "
+            + "specified by the user. \n"
```
Comment (jennibearduit) [inline src/main/java/seedu/track2gather/logic/commands/FindCommand.java:24]: Please remove extra space, thanks!

**Review by jennibearduit — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/track2gather/logic/commands/FindCommand.java (hunk 2 above)
Comment (luffingluffy) [reply src/main/java/seedu/track2gather/logic/commands/FindCommand.java]: Amended!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -26,33 +26,33 @@ Track2Gather is a **desktop app for contact tracing personnel at the [Ministry o
...
+    * **`schedule`** : Displays a filtered list of all persons who have not been called in the current SHN enforcement session.
 
     * **`scall`** : Updates a person as successfully called in the current SHN enforcement session.
 
     * **`fcall`** : Updates that a failed call was made to a person in the current SHN enforcement session.
     
     * **`help`** : Shows a message explaining how to access the help page.
 
-    * **`exit`** : Exits the app.
+    * **`exit`** : Exits the program.
```
Comment (jennibearduit) [inline docs/UserGuide.md:55]: What's the reason for changing "app" -> "program" haha

File: docs/UserGuide.md [hunk 4]
```diff
@@ -26,33 +26,33 @@ Track2Gather is a **desktop app for contact tracing personnel at the [Ministry o
...
+    * **`find`** : Finds person(s) based on the field specified by the user.
     
-    * **`sort`** : Sorts all persons in the persons list by certain fields.
+    * **`sort`** : Sorts the persons list based on the field(s) specified by the user.
     
-    * **`session`** : Starts a new SHN enforcement session with all persons set to 'not called'.
+    * **`session`** : Starts a new SHN enforcement session by resetting the call status indicators of all persons to ‘not called’.
 
-    * **`schedule`** : Shows a list of all persons who have not been called in the current SHN enforcement session.
+    * **`schedule`** : Displays a filtered list of all persons who have not been called in the current SHN enforcement session.
```
Comment (jennibearduit) [inline docs/UserGuide.md:47]: I prefer having "filtered" removed, not sure what the others think

File: src/main/java/seedu/track2gather/logic/commands/FCallCommand.java [hunk 5]
```diff
@@ -11,15 +11,14 @@
...
- * number of failed attempts.
+ * Updates that a failed call was made to a person in the current SHN enforcement session.
  */
 public class FCallCommand extends Command {
 
     public static final String COMMAND_WORD = "fcall";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-        + ": Indicates that a person was non-responsive when an attempt to contact failed\n"
+        + ": Updates that a failed call was made to a person in the current SHN enforcement session.\n"
```
[the remark this answers is not in the document]
Comment (jennibearduit) [reply src/main/java/seedu/track2gather/logic/commands/FCallCommand.java:21]: Never mind I think this is not an issue.

**Review by jennibearduit — COMMENTED** [review]
Some comments for consideration!

File: src/main/java/seedu/track2gather/logic/commands/FindCommand.java [hunk 6]
```diff
@@ -14,19 +14,17 @@
...
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons who fulfills the "
-            + "criteria specified by the user.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds person(s) based on the field "
+            + "specified by the user.\n"
             + "Parameters (indicate only 1): "
             + "[" + PREFIX_NAME + "NAME] "
-            + "[" + PREFIX_PHONE + "PHONE] "
+            + "[" + PREFIX_PHONE + "PHONE_NUMBER] "
             + "[" + PREFIX_CASE_NUMBER + "CASE NUMBER] "
```
Comment (wei2912) [inline src/main/java/seedu/track2gather/logic/commands/FindCommand.java:28]: Should be `CASE_NUMBER`

**Comment by wei2912** [comment]
Gonna approve now lol

**Review by wei2912 — APPROVED** [review]
LGTM!

**Pull request merged**